src/test/java/Homework19.java

```diff
@@ -1,43 +1,58 @@
 import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
 import org.openqa.selenium.WebElement;
 import org.testng.Assert;
+import org.testng.annotations.Optional;
+import org.testng.annotations.Parameters;
 import org.testng.annotations.Test;
 
 public class Homework19 extends BaseTest {
 
     @Test
-    public void deletePlaylist() throws InterruptedException {
+    @Parameters({"baseURL"})
+    public void deletePlaylist(@Optional String baseURL) throws InterruptedException {
         // Create a test case deletePlaylist() using @Test annotations
         // Use the reusable methods
         // Use @Parameters for passing baseUrl
         // Given: Navigate to "https://bbb.testpro.io/"
         // Step1: Log in with your credentials
+        if(baseURL != null) {
+            this.url = baseURL;
+        }
         login();
         // Step2: Choose a playlist
         selectPlaylist();
         // Step3: Delete the playlist
         pressDeletePlaylist();
         // Step4: Validate that the playlist is deleted "Deleted playlist"
-        Assert.assertEquals(getConfirmationPopupText(), "Deleted playlist");
+        Assert.assertTrue(getConfirmationPopupText().contains("Deleted playlist"));
     }
 
     private void selectPlaylist() throws InterruptedException {
         WebElement myPlaylist = driver.findElement(By.cssSelector(".playlist:nth-child(3)"));
         myPlaylist.click();
-        Thread.sleep(5000);
+        Thread.sleep(1000);
     }
 
     private void pressDeletePlaylist() throws InterruptedException {
+
         WebElement delPlaylist = driver.findElement(By.cssSelector(".btn-delete-playlist"));
         delPlaylist.click();
-        Thread.sleep(4000);
+
+        try {
+            WebElement confirmOkElement = driver.findElement(By.xpath("//button[contains(@class,'ok')]"));
+            confirmOkElement.click();
+        } catch (NoSuchElementException e) {
+            System.out.println("Confirm dialog was not present!");
+        }
+
+        // Thread.sleep(2000);
```

Comment:
I fixed the errors and the test is run, 

here is an example

![image](https://user-images.githubusercontent.com/104761482/200297618-89494303-365e-4bc3-a83c-54cfcff450a9.png)

> The test fails in some scenario. Apparently, when it fails, it is not finding the webElement by css selector ('div.success.show') though I can see in UI it does show up. I tried with different sleeping time but with no success.

I add `driver.manage().window().maximize();` in BaseTest

Use the maximize method of the Window interface:

![image](https://user-images.githubusercontent.com/104761482/200298928-0e7033cd-0c2d-433b-9636-633222d3c638.png)